orte/mca/rml/ofi/rml_ofi.h

```diff
@@ -40,6 +43,9 @@
 #define MULTI_BUF_SIZE_FACTOR 128
 #define MIN_MULTI_BUF_SIZE (1024 * 1024)
 
+#define SOCKADDR    "ofi-sockaddr"
+#define PSMXADDR    "ofi-psmxaddr"
```

Comment:
Seems odd to have a PSM-specific construct in this code...?  Isn't this supposed to be generic libfabric code?